src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -43,22 +43,9 @@ public override void MarkItemAsImported(DownloadClientItem downloadClientItem)
                 {
                     Proxy.SetTorrentLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
                 }
-                catch (DownloadClientException ex)
+                catch (DownloadClientException)
                 {
-                    if (ex.InnerException is HttpException && (ex.InnerException as HttpException).Response.StatusCode == HttpStatusCode.Conflict)
-                    {
-                        // category doesn't exist; add and try again
-                        Proxy.AddTorrentLabel(Settings.TvImportedCategory, Settings);
-                        Proxy.SetTorrentLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
-                    }
-                    else
-                    {
-                        throw;
-                    }
-                }
-                catch (Exception ex)
-                {
-                    _logger.Warn(ex, "Failed to set torrent label \"{0}\" for {1} in qBittorrent", 
+                    _logger.Warn("Failed to set post-import torrent label \"{0}\" for {1} in qBittorrent. Does the label exist?",
```

Comment:
My thought process is we are kind of expecting this as it happens when a user removes the label from their client after initially setting it. Stacktrace won't tell us anything we don't already know here and since it's just the warning I didn't want to clutter the log with unnecessary info. If it's an exception we aren't expecting, it will get thrown to DownloadEventHub as an error where the stacktrace will get shown.